test/Lsp.Tests/SemanticTokensDocumentTests.cs

```diff
@@ -126,31 +126,13 @@ public class ReturnDocumentTokensFromScratchForRangeData : TheoryData<Range, IEn
             public ReturnDocumentTokensFromScratchForRangeData()
             {
                 Add(
-                    new Range {
-                        Start = new Position {
-                            Line = 12,
-                            Character = 21,
-                        },
-                        End = new Position {
-                            Line = 14,
-                            Character = 27
-                        }
-                    },
+                    ((12, 21),(14, 27)),
```

Comment:
Hmm, C#9 certainly makes this more terse, but I'd have to hover over `Add` to find out that this is `Range` and `Position`. Personally I'd make that more explicit, but it's not my code-base.

Reply:
that's actually the implicit operators, but there is nothing stopping you from using which syntax makes sense.